config.py

```diff
@@ -24,6 +24,8 @@ class Config(object):
     JWT_BLACKLIST_ENABLED = True
     JWT_BLACKLIST_TOKEN_CHECKS = ["access", "refresh"]
     JWT_ACCESS_TOKEN_EXPIRES = 43200
+    ELASTICSEARCH_URL = os.environ.get("ELASTICSEARCH_URL", "http://localhost:9200")
```

Comment:
in case you add ssl/tls support for https protocal this call to connect to the ES server instance woud break

perhaps it would be safe to do this dynamically

```
import urlparse # e.g

domain = urlparse(request.META.get('HTTP_REFERER')).host # e.g 'http://domain_name.com'
ELASTICSEARCH_URL = os.environ.get("ELASTICSEARCH_URL", f"{domain}:9200")
```

NOTE: that this is just to give you an idea of what you need to do and not that the above pseudocode to be taken as the exact code to write